### ISSUE TITLE
line number of error message is wrong

### ISSUE THREAD

**Issue by bronze1man** [vlang/v#3026]
**V version:** 
**OS:** 
https://vlang.io/play

**What did you do?**
```v
fn fib(n int) int {
        if n <= 1 {
                return n
        }
        return fib(n - 1) + fib(n - 2)
}
mut run_num:=0

fn main() {
        for i := 0; i < 10; i++ {
                println(fib(i))
        }
        println(run_num)
}
```

**What did you expect to see?**
error at line 14
 
**What did you see instead?**
error at line 12

**Comment by sereg1k** [comment]
Also, https://github.com/vlang/v/issues/2655

**Comment by alexesprit** [comment]
Closed as a duplicate of #2655.
